enterprise/server/remote_execution/vfs/vfs_unix.go

```diff
@@ -130,10 +130,11 @@ func (vfs *VFS) Mount() error {
 			// Don't depend on `fusermount`.
 			// Disable fallback to fusermount as well, since it can cause
 			// deadlocks. See https://github.com/hanwen/go-fuse/issues/506
-			DirectMountStrict: true,
-			FsName:            "bbvfs",
-			MaxWrite:          fuse.MAX_KERNEL_WRITE,
-			EnableLocks:       true,
+			DirectMountStrict:  true,
+			DisableReadDirPlus: true,
```

Comment:
comment would be useful for this one

Reply:
I removed this. There's no perf difference now that we implement `Lookup` ourselves.